### ISSUE TITLE
I would like to limit the number of radio buttons being generated by surveyCreator in surveyjs library when I select radiogroup question type

### ISSUE THREAD

**Issue by hkibet** [surveyjs/survey-creator#4052]
### Are you requesting a feature, reporting a bug or ask a question?
I'm asking a question

### What is the current behavior?

The current behavious is that, when I select a radiogroup question type, it automatically brings six options: Three items and three plus icons.

### What is the expected behavior?
I would like to limit the number of radio buttons to 3, two items and one plus icon.

### How would you reproduce the current behavior (if this is a bug)?

You can reproduce the behaviour by choosing radiogroup question type

#### Provide the test code and the tested page URL (if applicable)
This is the url: https://surveyjs.io/survey-creator/examples/free-nps-survey-template/reactjs . If you choose radiogroup, you will get six items.

I have tried this: 

```
creator.toolbox
    .getItemByName("radiogroup")
    .json
    .choices = [
      { text: "Option 1", value: 1 },
      { text: "Option 2", value: 2 },
    ];

and 

PropertyGridEditorCollection.register({
    
    fit: function (prop) {
      return prop.type === "radiogroup";
    },
   
    getJSON: function (obj, prop, options) {
      console.log("obj", obj);
      return { type: "radiogroup", choices: ["Option 1"] };
    }
  });

but still they didn't work

```

**Comment by JaneSjs** [comment]
Hello @hkibet,
You can specify choices for the Radiogroup toolbox item using the [SurveyCreator.settings.toolbox.defaultJSON](https://github.com/surveyjs/survey-creator/blob/master/packages/survey-creator-core/src/creator-settings.ts#L103) settings. For instance, the following code sets the default number of preconfigured radiogroup items to 2:
```
SurveyCreator.settings.toolbox.defaultJSON.radiogroup = {
        choices: ["Item 1", "Item 2"]
};
```
Now, a newly added Radiogroup question appears as follows:
![image](https://user-images.githubusercontent.com/22372972/235630448-71645b96-6e41-4a85-9854-791bfedb1a98.png)

If you wish to additionally hide the None and Other items, you can handle the `survey.onShowingChoiceItem` function for the edited survey and hide these items from the design surface:
```
creator.onSurveyInstanceCreated.add(function(sender, options) {   
   if(options.reason == "designer") {
            options.survey.onShowingChoiceItem.add(function (sender, options){
              let stopItemList = [ 'none', 'other'];
              if(stopItemList.indexOf(options.item.id) > -1) {
                    options.visible = false;
                }
            });
   }
```
Now, a Radiogroup question appears as follows:
![image](https://user-images.githubusercontent.com/22372972/235631567-6d85c386-f6ee-423a-b1b1-299320918fb2.png)

Please feel free to reactivate this thread if you have further questions or require assistance.

**Comment by hkibet** [comment]
Thanks for reaching out.
 
I tried this:
`SurveyCreator.settings.toolbox.defaultJSON.radiogroup = {
        choices: ["Item 1", "Item 2"]
};`

And for some reason, there's no settings property in SurveyCreator class. I went ahead and tried this:
`creator.toolbox.defaultJSON.radiogroup = {
    choices: ["Item 1", "Item 2"]
  }`
And also there's no defaultJSON property in the creator.toolbox.
This:
`creator.onSurveyInstanceCreated.add(function(sender, options) {   
   if(options.reason == "designer") {
            options.survey.onShowingChoiceItem.add(function (sender, options){
              let stopItemList = [ 'none', 'other'];
              if(stopItemList.indexOf(options.item.id) > -1) {
                    options.visible = false;
                }
            });
   }`

really worked fine. I'm only having a challenge to change the name of the last add item to be something like, "Click on the add Icon to add another answer".
